### ISSUE TITLE
Marginal effect averaged across all response levels

### ISSUE THREAD

**Issue by JacobElder** [strengejacke/ggeffects#253]
When inputting a model for ggeffects that is ordinal logistic, it adds an additional column for response.level. Is there a way to collapse across all response levels and look at an average effect across each response level? For example, I have an interaction plot that looks like this for an ordinal logistic. Could I collapse across those 7 response levels and look at overall, average marginal effects collapsed across?

![000008](https://user-images.githubusercontent.com/16260312/167048699-2ad34a90-7f80-44e8-bf4b-15234523049c.png)

**Comment by strengejacke** [comment]
No, currently not possible. You *could* compute the summary/average manually, but `ggpredict()` returns values on the response scale, and I think due to Jensen's inequality, it's probably better to average predictions on the link-scale, than transform averages to response scale.
